### ISSUE TITLE
Files with numeric prefixes are sorted incorrectly.

### ISSUE THREAD

**Issue by Flamenco** [benweet/stackedit#1498]
See https://github.com/benweet/stackedit/pull/1497

I order my files by prepending a number.
```
1. Filename
2. Filename
10. Filename
20. Filename
```

The default collation does not use the `numeric` option, so my files get ordered like this:
```
1. Filename
10. Filename
2. Filename
20. Filename
```

**Comment by benweet** [comment]
Should be fixed in v5.14.

**Comment by Flamenco** [comment]
@benweet I have a better version of this that handles 1.25, 1.5, etc.  Interested?
